pkg/controller/worker/machines.go

```diff
@@ -41,6 +42,15 @@ const (
 	VolumeTypeScratch = "SCRATCH"
 )
 
+var (
+	// AllowedTypesIops are the volume types for which iops can be configured
+	// https://cloud.google.com/sdk/gcloud/reference/compute/disks/create#--provisioned-iops
+	AllowedTypesIops = []string{"pd-extreme", "hyperdisk-extreme"}
+	// AllowedTypesThroughput are the the volume types for which throughput can be configured
+	// https://cloud.google.com/sdk/gcloud/reference/compute/disks/create#--provisioned-throughput
+	AllowedTypesThroughput = []string{"hyperdisk-throughput"}
+)
+
```

Comment:
Consider defining these strings in the constants section (and put the consts in the slice)